src/scripts/farming/Farming.ts

```diff
@@ -1610,9 +1610,9 @@ class Farming implements Feature {
     }
 
     addPlotAuras(auraType: AuraType): number {
-        return this.plotList
+        return Math.max(0, this.plotList
```

Comment:
Why this `Math.max`?

Reply:
It remains from my original idea and followed Eugene's usage of the function. It is useless as of now.

Reply:
I don't think we should have it - it isn't needed for Micle to work and would prevent negative auras being summed if we ever add one.